src/DynamoCore/Extensions/LinterExtensionBase.cs

```diff
@@ -19,6 +19,7 @@ internal abstract class LinterExtensionBase : IExtension
     {
         private const string NODE_ADDED_PROPERTY = "NodeAdded";
         private const string NODE_REMOVED_PROPERTY = "NodeRemoved";
+        private const string NODE_MODIFIED_PROPERTY = "Modified";
```

Comment:
Minor comment: Shouldn't this be `NodeModified` if following the naming convention here?